tower-balance/src/p2c/service.rs

```diff
@@ -197,14 +195,7 @@ where
                 let bload = self.ready_index_load(bidx);
                 let ready = if aload <= bload { aidx } else { bidx };
 
-                trace!(
-                    "load[{}]={:?}; load[{}]={:?} -> ready={}",
-                    aidx,
-                    aload,
-                    bidx,
-                    bload,
-                    ready
-                );
+                trace!({ a.idx = aidx, a.load = ?aload, b.idx = bidx, b.load = ?bload, ready = ?ready }, "choosing by load");
```

Comment:
Oh, one final thing, extremely minor: I notice the way we name the fields for indices in this file is kinda inconsistent: on line 174, we drop the `idx` from the variable names but refer to the repaired index as `i`, while here, we use `a.idx` and `b.idx`, and on line 288 we have a field called `index`. Maybe we should make these more consistent?